### PR TITLE
Cu 86b15rjqy fix icons

### PR DESCRIPTION
**Pull request by whamester**
I removed the user avatars and I'll complete that in the next ticket
Screenshot 2024-07-09 at 4 10 36 PM
Screenshot 2024-07-09 at 4 10 42 PM

- create challenge ui updated
- all png images that are not in app icons are removed
- TODO: replace challenges icons on thursday

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b15rjqy Fix Icons](https://app.clickup.com/t/86b15rjqy)

**Review by ReinhardtBotha — APPROVED** [review]
Nice! 👌🔥✅🥩☠️💯🐶👍👏🏆

File: src/components/challenges/ChallengeAvatarCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            Icon = Icon1;
+          }
+          if (index == 2) {
+            Icon = Icon3;
+          }
+          return (
+            <TouchableWithoutFeedback
+              key={index}
+              onPress={() => {
+                console.log(index);
```
Comment (ReinhardtBotha) [inline src/components/challenges/ChallengeAvatarCarousel.tsx]: maybe we can remove this console log

**Review by zeckkimon — APPROVED** [review]

**Review by Victorportus — APPROVED** [review]

File: src/components/challenges/ChallengeAvatarCarousel.tsx (hunk 1 above)
Comment (whamester) [reply src/components/challenges/ChallengeAvatarCarousel.tsx]: done

**Pull request merged**